File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -30,7 +30,6 @@ using System.IO;
 
 {% endhighlight %}
 
-
 Add the following method in the ExportService class
```

Comment:
Add the following code in the ExportService.cs file.

Reply:
modified the code